src/backends/policy_engine/souffle/sql_policy.dl

```diff
@@ -0,0 +1,49 @@
+//-----------------------------------------------------------------------------
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-----------------------------------------------------------------------------
+#ifndef SRC_BACKENDS_POLICY_ENGINE_SOUFFLE_SQL_POLICY_DL_
+#define SRC_BACKENDS_POLICY_ENGINE_SOUFFLE_SQL_POLICY_DL_
+
+#include "src/analysis/souffle/taint.dl"
+
+// OK: SELECT 42, 'foobar', null, true
+// OK: SELECT gender FROM people
+// FAIL: SELECT SSN FROM people
+//
+
+// Default owner associated with all tags related to SQL queries. 
+#define SQL_OWNER "sql"
```

Comment:
Currently, I'm defining the owning `Principal` as `"UNKNOWN"` because I don't have the context to know I'm in a SQL context in the lowering code: https://github.com/google-research/raksha/pull/534/files#diff-d8cf2a91dae5d2e5a25812afae3b80891cd31b3d5fab5806d169ca539b12fb7dR49

Reply:
Either works. I will leave it as "sql" just to help debugging easier? Also, this is a macro that could be changed?